src/components/_classes/component/Component.js

```diff
@@ -7,12 +7,13 @@ import isMobile from 'ismobilejs';
 import Formio from '../../../Formio';
 import * as FormioUtils from '../../../utils/utils';
 import Validator from '../../../validator/Validator';
+import TableModule from 'quill-table';
 import Templates from '../../../templates/Templates';
 import { fastCloneDeep, boolValue } from '../../../utils/utils';
 import Element from '../../../Element';
 import ComponentModal from '../componentModal/ComponentModal';
 const CKEDITOR = 'https://cdn.form.io/ckeditor/16.0.0/ckeditor.js';
-const QUILL_URL = 'https://cdn.form.io/quill/1.3.7';
+const QUILL_URL = 'https://cdn.quilljs.com/2.0.0-dev.2';
```

Comment:
seems risky to include a "dev" build of quill into our renderer.

Reply:
@travist 
This ``dev-2`` is the most stable from their ``devs``.
Unfortunately they still haven't released 2nd version and it is the only way how to provide tables for users.

Reply:
Is there any way we could lazy-load the table module?

Reply:
@travist no, didn't found any possibility to lazy load module.

Reply:
@travist Made it as option for textarea component.